src/protocol/eas/eas.ts

```diff
@@ -0,0 +1,459 @@
+  const decoded = decodeTransactionInput(transaction.input, ABIs.EAS);
+
+  switch (decoded.name) {
+    case 'attest': {
+      const { schema, data } = decoded.args[0];
+      const { recipient } = data;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          schema: {
+            type: 'emphasis',
+            value: schema,
+          },
+          recipient: {
+            type: 'address',
+            value: recipient,
+          },
+          attested: {
+            type: 'contextAction',
+            value: 'ATTESTED',
+          },
+        },
+        summaries: {
+          category: 'PROTOCOL_1',
+          en: {
+            title: 'EAS',
+            default: recipient
+              ? '[[from]] [[attested]] to [[recipient]] with schema [[schema]]'
+              : '[[from]] [[attested]] with schema [[schema]]',
+          },
+        },
+      };
+      return transaction;
+    }
+
+    case 'attestByDelegation': {
+      const { schema, attester, data } = decoded.args[0];
+      const { recipient } = data;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          attester: {
+            type: 'address',
+            value: attester,
+          },
+          schema: {
+            type: 'emphasis',
+            value: schema,
+          },
+          recipient: {
+            type: 'address',
+            value: recipient,
+          },
+          attested: {
+            type: 'contextAction',
+            value: 'ATTESTED',
+          },
+        },
+        summaries: {
+          category: 'PROTOCOL_1',
+          en: {
+            title: 'EAS',
+            default: recipient
+              ? '[[attester]] [[attested]] to [[recipient]] with schema [[schema]] by delegation via [[from]]'
+              : '[[attester]] [[attested]] with schema [[schema]] by delegation via [[from]]',
+          },
+        },
+      };
+      return transaction;
+    }
+
+    case 'multiAttest': {
+      const schemas = decoded.args[0].length;
+      const count = decoded.args[0].map((v) => v.data).flat().length;
+
+      transaction.context = {
+        variables: {
+          from: {
+            type: 'address',
+            value: transaction.from,
+          },
+          schemas: {
+            type: 'emphasis',
+            value: schemas.toString(),
+          },
+          count: {
+            type: 'emphasis',
+            value: count.toString(),
+          },
+          attested: {
+            type: 'contextAction',
+            value: 'ATTESTED',
+          },
+        },
+        summaries: {
+          category: 'PROTOCOL_1',
+          en: {
+            title: 'EAS',
+            default:
+              '[[from]] [[attested]] [[count]] times with [[schemas]] schemas',
```

Comment:
@datadanne Using numbers for this sentence was a nice approach

Reply:
If we made the unit change above:
```suggestion
              '[[from]] [[attested]] [[count]] with [[schemas]] schemas',
```